crates/polars-core/src/series/implementations/list.rs

```diff
@@ -189,6 +189,55 @@ impl SeriesTrait for SeriesWrap<ListChunked> {
         self.0.has_validity()
     }
 
+    #[cfg(feature = "group_by_list")]
+    fn unique(&self) -> PolarsResult<Series> {
+        if !self.inner_dtype().is_numeric() {
+            polars_bail!(opq = unique, self.inner_dtype());
+        }
+        // this can called in aggregation, so this fast path can be worth a lot
```

Comment:
typo, should say "this can **be** called"

Reply:
(:D)

Reply:
Yes, good catch!